### DISCUSSION TITLE
Afew images in baseline_table.dat hasn't any value.

### DISCUSSION THREAD

**Discussion by AlshimaaY** [gmtsar/gmtsar#882] [Q&A] [answered]
First massage
Reading in 6768 lines of info from XML...
Writing 17 lines for orbit...
Not an well formatted XML file...-1, -1
Unable to find designated string...after 0 searches..
NO orbit coverage in the selected file...

second massage
interpolation point outside of data constraints, 16127852.071749 2829600000.000000 2829600000.000000
......master LED file S1_20201013_ALL_F1.LED 
.........aligned LED file S1_20230718_ALL_F1.LED 

in step preproc_batch_tops.csh data.in dem.grd 1.
this occurs in image in 2023. 
please, help me to solve this problem.

**Comment by Xiaohua-Eric-Xu** [comment, marked as answer]
This is probably related to this issue here: https://github.com/gmtsar/gmtsar/discussions/629

**Reply by Xiaohua-Eric-Xu** [reply]
You may also choose to delete the blank lines in the EOF file to get rid of the error.

**Reply by AlshimaaY** [reply]
Thank you very much. It runs with me.
I have results until 2021, and I want to make new analyses in 2022 and 2023. How can I merge the old results with the new results from 2022 and 2023?

**Reply by Xiaohua-Eric-Xu** [reply]
Sorry for the late response, you could produce a new data.in file, then keep the supermaster dates and the new dates. Run preprocessing at a new place and move the products to the original raw directory. Then you can run select_pairs.csh again and diff the product from previous intf.in to figure out what new interferograms to produce. Produce them, merge them, unwrap them and re-run the sbas step to produce a new time-series.
